dnf/base.py

```diff
@@ -665,6 +669,7 @@ def do_transaction(self, display=()):
 
         if not self.transaction:
             if self._group_persistor:
+                self._trans_success = True
```

Comment:
Unless trans abbreviation is widely used in DNF, I'd prefer full name: self._transaction_success